docs/mergeTreeClustering.md

```diff
@@ -0,0 +1,49 @@
+# Merge Tree Clustering example
+
+![Merge Tree Clustering example Image](https://topology-tool-kit.github.io/img/gallery/mergeTreeClustering.jpg)
+
+## Pipeline description
+This example first loads an ensemble of scalar fields inside a single file from disk.
+Then, the [FTMTree](https://topology-tool-kit.github.io/doc/html/classttkFTMTree.html) is computed on each scalar field for the Join Tree and the Split Tree.
+
+All these trees are passed to [MergeTreeClustering](https://topology-tool-kit.github.io/doc/html/classttkMergeTreeClustering.html) to compute a clustering of merge trees. Each input is considered as a tuple consisting of the Join Tree and the Split Tree of the corresponding scalar field. Each centroid is also a tuple of this kind and a distance between two tuples is the distance between their Join Tree plus the distance between their Split Trees.
```

Comment:
"computes a clustering of merge trees"
->
"computes a clustering in the metric space of merge trees"

Reply:
Done in the next commit!